tests/torch/fx/test_fast_bias_correction.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+import pytest
+import torch
+from torch._export import capture_pre_autograd_graph
+
+from nncf.common.factory import NNCFGraphFactory
+from nncf.experimental.torch.fx.quantization.quantize_model import prepare_for_quantization
+from nncf.quantization.algorithms.fast_bias_correction.torch_fx_backend import FXFastBiasCorrectionAlgoBackend
+from nncf.torch.dynamic_graph.patch_pytorch import disable_patching
+from nncf.torch.nncf_network import NNCFNetwork
+from tests.post_training.test_templates.test_fast_bias_correction import TemplateTestFBCAlgorithm
+from tests.torch.fx.graph_manager import get_bias_value
+from tests.torch.fx.graph_manager import is_node_with_bias
+
+
+class TestTorchFXFBCAlgorithm(TemplateTestFBCAlgorithm):
+    @staticmethod
```

Comment:
Please implement a base class as it done in torch tests

Reply:
I'm sorry I didnt really get the suggestion, the template class is already implemented right?

Reply:
Yes, I was wrong. Please inherit `TestTorchFXFBCAlgorithm` in `TestTorchFXCudaFBCAlgorithm` and keep common methods (like get_backend_model and etc) in `TestTorchFXFBCAlgorithm`

Reply:
Oh, my mistake there, I will get those changes back and refactor the code.

Reply:
Done!